FirebaseMessaging/Tests/IntegrationTests/FIRMessagingTokenRefreshTests.swift

```diff
@@ -35,14 +34,16 @@
 
   class FIRMessagingTokenRefreshTests: XCTestCase {
     var app: FirebaseApp!
-    var messaging: Messaging?
+    var messaging: Messaging!
```

Comment:
Is the force unwrap removable by changing the assignment below to a `guard let`?

Reply:
No, you can only remove it if you either assign a value when it's declared or add a designated initializer to initialize it.